db/stats.py

```diff
@@ -1,4 +1,5 @@
-from db import create_cursor, commit, cache
+import db
+import db.cache
```

Comment:
There's actually a flaw in this module. See https://bitbucket.org/metabrainz/musicbrainz-server/pull-requests/1644/:

> The cache namespace key is stored in redis as opposed to memcached so that it doesn't get removed by any LRU algorithm.

*(we don't do that)*